setup.cfg

```diff
@@ -44,9 +44,9 @@ install_requires =
     scikit-image>=0.17.2
     scipy>=1.6.0,<1.10.0
     spherical-geometry>=1.2.22
-    stcal>=1.3.5
-    stdatamodels>=1.3.1
-    stpipe>=0.4.5
+    stcal>=1.3.5,<1.4.0
+    stdatamodels>=1.3.1,<1.4.0
+    stpipe>=0.4.5,<1.0.0
```

Comment:
Do we know that the next release of stpipe will jump to 1.0.0, as opposed to 0.5.0?